R/retro_wrapper.R

```diff
@@ -213,17 +212,19 @@ retro_wrapper <- function(mydir,  model_settings) {
       label = "RetroMohnsrho",
       escape = FALSE,
       caption = paste(
-        "The average relative bias of retrospective estimates",
+        "The magnitude of retrospective pattern",
         "(Mohn's rho; Mohn, 1999) given the removal of",
         xfun::numbers_to_words(max(abs(model_settings$retro_yr))),
         "years of data for",
         "fishing intensity (\\emph{F}),",
         "fraction unfished (Figure \\ref{fig:RetroFractionunfished}),",
         " recruitment, and",
         "spawning stock biomass (\\emph{SSB}; Figure \\ref{fig:RetroSsb}).",
-        "Columns are derivations of Mohn's rho used by the",
-        "Alaska Fisheries Science Center (AFSC),",
-        "suggested by Hurtado-Ferro et al. (2015), and",
+        "Columns are a derivation of Mohn's rho used by the",
```

Comment:
Thanks for these additions @brianlangseth-NOAA. I now read this list structure as
* Column is a derivation of Mohn's rho used by the Alaska Fisheries Science Center (AFSC)
* Column is suggested by Hurtado-Ferro et al. (2015)
* Column is as originally described in Mohn (1999)
* Column is a derivation of Mohn's rho (Woods Hole Mohn's rho) used by the Northeast Fisheries Science Center (NEFS)
Where before the precursor to each list item was "Column is derivation of Mohn's rho used by ...". Perhaps you can restructure this a bit so that each list item can be read as a sentence by itself with the "precursor"? Let me know if I am not making sense here.

Reply:
Im tracking with you @kellijohnson-NOAA. I restructured your commented section with commit 0597e90, making it more clear that there are three columns, 

- Column is a derivation of Mohn's rho used by the Alaska Fisheries Science Center (AFSC) as suggested by Hurtado-Ferro et al. (2015),
- Column is as originally described in Mohn (1999)
- Column is a derivation of Mohn's rho (Woods Hole Mohn's rho) used by the Northeast Fisheries Science Center (NEFS)

The prescursor is more clearly articulated too. My change extends the number of characters on a single line, which is what I originally tried to avoid by separating out the AFSC and Hurtado-Ferro, and Woods Hole and NEFSC statements.

Reply:
Sorry @brianlangseth-NOAA I didn't notice that there were not commas at the end of each line my brain just saw the comma after the " mark. I think we can shorten the first bullet to 
```
a derivation of Mohn's rho (Hurtado-Ferro et al., 2015) used by the Alaska Fisheries Science Center,
```
and get rid of the acronym for the NEFS. I was thinking about it and there is the potential for the acronym to already be defined in the text, and thus, we do not want to define it again.
Is there a citation for "Wood Hole Mohn's rho"?

Reply:
@kellijohnson-NOAA Are you suggesting I remove my NEFS abbreviation because its possibly different than the standard NEFSC abbreviation? That was a mistake in my reply - its NEFSC in the code. 

I clarify because the NEFSC and AFSC also define the column headers (AFSC, Mohn, NEFSC) in the table. The order of the descriptions make it fairly clear which column is which so Im fine removing but if we remove the NEFSC abbreviation we should probably also remove the AFSC abbreviation.

Reply:
No @brianlangseth-NOAA I honestly forgot that those were the column headers. So your suggestion of NEFSC is correct and add (AFSC) back to the end of my suggestion to make
```
a derivation of Mohn's rho (Hurtado-Ferro et al., 2015) used by the Alaska Fisheries Science Center (NEFSC),
```
Sorry I am making this so difficult.

Reply:
Changes made in 2d8604f. I added a reference for Woods-Hole rho (Legault 2009) both in the caption to the .tex table and with a link in the description of the function.

Reply:
@kellijohnson-NOAA Reminder: this pull request is still pending. Let me know if you are waiting on me for action.